tool/src/main.rs

```diff
@@ -54,12 +58,24 @@ fn main() {
         tas = Tas::new().unwrap();
         println!("TAS created successfully.");
     }
-
-    let script_file = env::args().skip(1).next().unwrap_or("tas.lua".to_string());
+    let script_file = if env::args().len() == 1 {
+        if std::env::current_dir().unwrap().join("tas.lua").metadata().is_err() {
+            let cur_dir = std::env::current_dir().unwrap();
+            let cur_dir = cur_dir.to_str().unwrap();
+            let result = nfd::open_file_dialog(Some("lua"), Some(cur_dir)).unwrap_or_else(|e| { panic!(e) });
+            match result {
+                Response::Okay(file_path) => Path::new(&file_path).to_str().unwrap().to_string(),
+                Response::OkayMultiple(_) => unreachable!("Multiple files selected."),
+                Response::Cancel => panic!("Cancelled file selection.")
+            }
+        } else {
+            std::env::current_dir().unwrap().join("tas.lua").to_str().unwrap().to_string()
```

Comment:
As I said before, `Path::new` / `File::open` are looking in the current_dir by default, so no need to manually join it.